src/mobile-sdk-core/UIError.ts

```diff
@@ -2,7 +2,7 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { ResponseError } from "@itwin/core-frontend/lib/cjs/request/Request";
+import { HttpResponseError } from "@itwin/core-frontend/lib/cjs/request/Request";
```

Comment:
really should not be importing from cjs like this for an internal api

Reply:
Do you have another suggestion that lets us provide that information to SYNCHRO Field?

Reply:
It's not pretty but we could parse the .message property of the Error knowing how it's populated by HttpResponseError.

Reply:
core can consider making it public, but it may be better to drop the `instanceof` check. A little ugly but instanceof has its own issues...

Reply:
i was going to suggest you guys copy the impl yourself, 
we def dont want to expose this as a public api either
given your tight coupling with the library and pinning the version down to the exact ver this can be ignored for now i think

Reply:
If I just remove the code entirely, then message contains a string that includes the status code and the response text, so maybe that's the best solution, even with our close coupling to the specific iTwin version.

Reply:
I removed the special handling. The UIError's message will still include the information.